arviz/stats/stats.py

```diff
@@ -1258,17 +1265,16 @@ def summary(
             metrics_names_ = metrics_names_[4:]
         metrics.extend(metrics_)
         metric_names.extend(metrics_names_)
-    if include_circ:
-        metrics.extend((circ_mean, circ_sd, circ_hdi_lower, circ_hdi_higher, circ_mcse))
-        metric_names.extend(
-            (
-                "circular_mean",
-                "circular_sd",
-                "circular_hdi_{:g}%".format(100 * alpha / 2),
-                "circular_hdi_{:g}%".format(100 * (1 - alpha / 2)),
-                "circular_mcse",
-            )
-        )
+
+    if circ_var_names:
+
+        if kind != "diagnostics":
+            for metric, circ_stat in zip(
+                metrics[:5], (circ_mean, circ_sd, circ_hdi_lower, circ_hdi_higher, circ_mcse)
```

Comment:
Whats 5 here? Would add comment for the magic number

Reply:
Agree about the comment. 
The 5 is related to the "stats" columns, the diagnostics are computed the same for linear and circular variables. Actually, we have to think if the diagnostics also need to be changed for circular variables (not part of this PR).